.travis.yml

```diff
@@ -21,58 +25,161 @@ matrix:
       os: osx
       env:
         - WebPackiOS="12.0"
+        - Type="VanillaJS"
       osx_image: xcode10.0
       language: node_js 
       node_js: "8"
       jdk: oraclejdk8
-      script: cd demo && npm run build.plugin && npm i && tns build ios --env.uglify && cd ../demo-angular && npm i && tns build ios --env.uglify --env.aot
+      before_script: pod repo update
```

Comment:
there are no pods used in the camera plugin, so you can remove `pod repo update` steps